unattended_installer/install_functions/checks.sh

```diff
@@ -178,8 +178,8 @@ function check_dist() {
         notsupported=1
     fi
     if [ "${DIST_NAME}" == "ubuntu" ]; then
-        if [ "${DIST_VER}" == "16" ] || [ "${DIST_VER}" == "18" ] ||
-           [ "${DIST_VER}" == "20" ] || [ "${DIST_VER}" == "22" ]; then
+        if  [ "${DIST_VER}" == "16" ] || [ "${DIST_VER}" == "18" ] ||
```

Comment:
Why 2 spaces?

Reply:
It had a bad indentation https://github.com/wazuh/wazuh-packages/pull/1742/commits/e2e272599a162ddbb87d6fc8462e62ba849d0990